pkg/operator/metrics/metrics.go

```diff
@@ -118,11 +119,12 @@ func (mc *Calculator) metricsLoop() {
 
 	mc.log.Info("calculating metrics for all CredentialsRequests")
 
-	_, ccoDisabled, _, err := utils.GetOperatorConfiguration(mc.Client, mc.log)
+	mode, _, err := utils.GetOperatorConfiguration(mc.Client, mc.log)
 	if err != nil {
 		mc.log.WithError(err).Error("failed to determine whether CCO is disabled")
 		return
 	}
+	ccoDisabled := mode == operatorv1.CloudCredentialsModeManual
```

Comment:
Also not something to change now, but we should make an effort in the future to remove the "disabled" concept from the CCO and replace it with "manual mode".